src/node_dotenv.cc

```diff
@@ -130,50 +135,81 @@ void Dotenv::ParseContent(const std::string_view input) {
 
   while (!content.empty()) {
     // Skip empty lines and comments
+    // Example:
+    //   # This is a comment
+    //
+    //   KEY=value
     if (content.front() == '\n' || content.front() == '#') {
       auto newline = content.find('\n');
```

Comment:
If content.front() is \n this will always return 0 (the first character). I wonder why we did it like this...

Reply:
We can simply replace line 144 with `if (content.front() == '\n')` and remove the content.find('\n') check I guess...